app/code/Magento/Newsletter/Model/Subscriber.php

```diff
@@ -353,11 +353,7 @@ public function isStatusChanged()
      */
     public function isSubscribed()
     {
-        if ($this->getId() && $this->getStatus() == self::STATUS_SUBSCRIBED) {
-            return true;
-        }
-
-        return false;
+        return $this->getId() && $this->getStatus() == self::STATUS_SUBSCRIBED;
```

Comment:
Would be great also replace == to === if possible there

Reply:
I thought about that but was trying to keep the refactor focused. I'm happy to tighten the equality if desired.